internalWebhook.go

```diff
@@ -47,6 +47,14 @@ func (v2 *RegistryV2) GetUntil() time.Time {
 	return v2.Registration.Expires
 }
 
+func (v2 *RegistryV2) GetWebhookType() {
```

Comment:
assuming this is something you're working on for later?

Reply:
whops, good catch!